charts/firefly/scripts/ff-db-migrations.sh

```diff
@@ -19,8 +19,6 @@
 # Install deps
 apk add postgresql-client curl jq
 
-echo "Provided connection string: '${PSQL_URL}'"
```

Comment:
Can we keep the `echo` but only do it if a `DEBUG` env var is set. I agree with the spirit of this print statement since its useful when we are doing chart development / debugging the migrations job but obviously we don't want this on by default going forward for production scenarios.

Reply:
I did see your note on that but since the DB creds are so sensitive, I wonder if it's best to remove them outright.

Reply:
Yeah fair enough, if we are debugging something with migrations we'll likely be developing against the chart and can edit the script temporarily if we need to.